packages/uikit-react-native/src/components/ChannelInput/index.tsx

```diff
@@ -98,11 +101,18 @@ const ChannelInput = (props: ChannelInputProps) => {
 
   const mentionAvailable =
     sbOptions.uikit.groupChannel.channel.enableMention && channel.isGroupChannel() && !channel.isBroadcast;
-
   const inputKeyToRemount = GET_INPUT_KEY(mentionAvailable ? mentionedUsers.length === 0 : false);
 
   const [inputHeight, setInputHeight] = useState(styles.inputDefault.height);
 
+  const fontStyle = useMemo(() => {
+    if (!typography.body3.fontSize) return typography.body3;
+    // NOTE: iOS does not support textAlignVertical, so we should adjust lineHeight to center the text in multiline TextInput.
+    return { ...typography.body3, lineHeight: typography.body3.fontSize * 1.275, textAlignVertical: 'center' };
```

Comment:
Generally, to obtain accurate height on iOS, a [specific formula](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/TextLayout/Tasks/StringHeight.html) is required. However, since it's not executable in JavaScript, the magic number `1.2` is used(0.075 is offset for custom input height).

This number aligns reasonably well with the default system font.
However, if the customer employs a different font, the lineHeight must be adjusted directly, leading to the opening of the style props.